site/content/configuration/access-control.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Deploy an Access Control policy
+weight: 900
+toc: true
+docs: DOCS-000
+---
+
+This topic describes how to apply and update an Access Control policy with F5 NGINX Ingress Controller.
+
+It demonstrates this using an example application and a [VirtualServer custom resource]({{< ref "/configuration/virtualserver-and-virtualserverroute-resources.md" >}})
+
+---
+
+## Before you begin
+
+You should have a [working NGINX Ingress Controller]({{< ref "/installation/installing-nic/installation-with-helm.md" >}}) instance.
+
+For ease of use in shell commands, set two shell variables:
+
+1. The public IP address for your NGINX Ingress Controller instance.
+
+```shell
+IC_IP=<ip-address>
+```
+
+2. The HTTP port of the same instance
+
+```shell
+IC_HTTP_PORT=<port number>
+```
+
+---
+
+## Deploy the example application
+
+Create the file *webapp.yaml* with the following contents:
+
+{{< ghcode "https://raw.githubusercontent.com/nginxinc/kubernetes-ingress/refs/heads/main/examples/custom-resources/access-control/webapp.yaml" >}}
+
+Apply it using *kubectl*:
+
+```shell
+kubectl apply -f webapp.yaml
+```
+
+---
+
+## Deploy an Access Control policy
```

Comment:
```suggestion
## Deploy a Policy to deny application access
```

Reply:
The focus of the document as a whole is to show how to deploy an Access Control policy: the deny and allow steps existence in sequence to demonstrate this, so I don't think it makes sense to make the policy the focus nor is it necessary to be this explicit.

Reply:
You're using the same text as the title of the document in a sub-heading, which is confusing for users. If the broad goal of the doc is to deploy an access control policy, and I complete this section, am I now done with the doc? Why are there more sections? It makes more sense to me to have the sub-headings accurately reflect the expected outcome of completing the steps in the section.